cnsctl/cmd/ova/cleanup.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package ova
+
+import (
+	"fmt"
+	"github.com/spf13/viper"
+	"os"
+	"github.com/spf13/cobra"
+)
+
+// cleanupCmd represents the cleanup command
+var cleanupCmd = &cobra.Command{
+	Use:   "cleanup",
+	Short: "Identifies orphan volume attachment CRs and deletes them",
+	Long:  "Identifies orphan volume attachment CRs and deletes them",
+	Run: func(cmd *cobra.Command, args []string) {
+		validateCleanupFlags()
+
+		if len(args) != 0 {
```

Comment:
This check is not provided for `ov cleanup` command. Can we keep it consistent across `ov` & `ova` cleanup commands, or remove it from here?